libvips/foreign/jxlsave.c

```diff
@@ -87,6 +88,10 @@ typedef struct _VipsForeignSaveJxl {
 	gboolean lossless;
 	int Q;
 
+	/* Set on API fail.
+	 */
+	gboolean error;
```

Comment:
This seems to be unused, as it's set but never checked.

Reply:
Nice catch, I'll add some checks.